locales/en/bento.ftl

```diff
@@ -5,7 +5,6 @@
 toolbar-app-picker-trigger-title = { -brand-mozilla } apps and services
 toolbar-app-picker-product-vpn = { -brand-mozilla-vpn }
 toolbar-app-picker-product-relay = { -brand-relay }
-toolbar-app-picker-product-pocket = { -brand-pocket }
```

Comment:
Is there a reason to keep the brand definition?
https://github.com/mozilla/blurts-server/blob/main/locales/en/app.ftl#L15

We should probably remove also the Firefox Account one, together with `manage-fxa`, but that's a different topic.